monkey/agent_plugins/exploiters/snmp/src/snmp_exploiter.py

```diff
@@ -0,0 +1,124 @@
+from logging import getLogger
```

Comment:
Based on the coverage report, it looks like we're missing a test where the exploit client raises an exception.